CHANGES.md

```diff
@@ -28,6 +28,27 @@ API Changes:
 * [#1498](https://github.com/ckeditor/ckeditor-dev/issues/1498) : Added new method 'getClientRects()' to CKEDITOR.dom.range, which returns list of rects for each selected element.
 * [#1993](https://github.com/ckeditor/ckeditor-dev/issues/1993): Added [`CKEDITOR.tools.throttle`](http://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR_tools.html#method-throttle) function.
 
+Other Changes:
+
+* Updated [SCAYT](https://ckeditor.com/cke4/addon/scayt) (Spell Check As You Type) and [WebSpellChecker](https://ckeditor.com/cke4/addon/wsc) (WSC) plugins:
+	* Language Dictionaries Updates - Added support of the Uzbek Latin language.
+	* No Longer Supported Languages - Manx - Isle of Man (gv_GB) and Interlingua (ia_XR) languages are no longer available as an additional language.
+	* Extended and Improved Language Dictionaries - Georgian and Swedish languages dictionary has been enlarged. Added missing word _"Ensure"_ to the American/British/Canada English language
+	* Fixed: [#141](https://github.com/WebSpellChecker/ckeditor-plugin-scayt/issues/141) SCAYT plugin for CKEditor: SCAYT throws Uncaught Error: Error in RangyWrappedRange module: createRange(): Parameter must be a Window object or DOM node
+	* Fixed: [#153](https://github.com/WebSpellChecker/ckeditor-plugin-scayt/issues/153) SCAYT plugin for CKEditor: Chrome. Correcting word in widget moves focus to anothereditable
+	* Fixed: [#155](https://github.com/WebSpellChecker/ckeditor-plugin-scayt/issues/155) SCAYT plugin for CKEditor: SCAYT throw an error on IE8 and doesn't work
+	* Fixed: [#156](https://github.com/WebSpellChecker/ckeditor-plugin-scayt/issues/156) SCAYT plugin for CKEditor: SCAYT doesn't seem to work on IE10
+	* Fixed: SCAYT plugin for CKEditor: After some text is drag & dropped, markup is not refreshedfor grammar problems
+	* Fixed: WSC plugin for CKEditor: Request to FastCGI fails when user tries to replace word withnon-English characters with a proper suggestion
+	* Fixed: SCAYT Core: CTRL+Z removes focus in FireFox
+	* Enchancments: Grammar Support for Default Languages
+	* Enchancments: New Application Source URL in SCAYT plugin for CKEditor
```

Comment:
"plugin for CKEditor" makes little sense as this is in CKEditor changelog so what else editor should it be related to? :)